api_app/event_grid/helpers.py

```diff
@@ -0,0 +1,34 @@
+import logging
+from azure.eventgrid import EventGridEvent
+from azure.eventgrid.aio import EventGridPublisherClient
+from azure.core.credentials import AzureKeyCredential
+from models.domain.airlock_request import AirlockRequest
+from core import config
+
+
+async def _publish_event(event: EventGridEvent, topic_key: str, topic_endpoint: str):
+    credential = AzureKeyCredential(topic_key)
+    client = EventGridPublisherClient(topic_endpoint, credential)
+
```

Comment:
add logs

Reply:
The logging happens is the function that calls that one